apps/docs/content/docs/ui/(integrations)/openapi/configurations.mdx

```diff
@@ -138,6 +138,66 @@ void generateFiles({
 });
 ```
 
+### Add Generated Comment
+
+Add a comment to the top of generated files indicating they are auto-generated.
+
+| Value    | Description                           |
+| -------- | ------------------------------------- |
+| `true`   | Adds a standardized comment (default) |
+| `false`  | No comment is added                   |
+| `string` | Adds the provided custom comment      |
+
+```ts
+import { generateFiles } from 'fumadocs-openapi';
+
+void generateFiles({
+  input: ['./petstore.yaml'],
+  output: './content/docs',
+  // Add default comment
+  addGeneratedComment: true,
+
+  // Or provide a custom comment
+  addGeneratedComment: 'Custom auto-generated comment',
+
+  // Or disable comments
+  addGeneratedComment: false,
+});
+```
+
+Default standardized comment content:
+
+```
+This file was auto-generated by Fuma Docs.
+Do not edit this file directly. Any changes should be made by running the generation command again.
+```
+
+This content is added as MDX comments below the frontmatter:
+
+{/* prettier-ignore-start */}
```

Comment:
This is required otherwise the example comment gets reformatted as

```
{/_ This file was auto-generated by Fuma Docs.
Do not edit this file directly. Any changes should be made by running the generation command again. _/}
```

and it renders with the underscores which is incorrect.